tests/integration/conftest.py

```diff
@@ -37,3 +49,13 @@ def test_snap(local_test_snap_path):
     # teardown
     if is_snap_installed("test-snap"):
         wait_for(snap_http.remove)("test-snap")
+
+
+@pytest.fixture
+def hello_world_snap_declaration_assertion() -> (str, Dict[str, str]):
+    path = "tests/integration/assets/hello_world_snap_declaration.assert"
+    with open(path, "rb") as f:
+        yield (f.read().decode(), HELLO_WORLD_SNAP_DECLARATION_ASSERTION)
```

Comment:
what's the purpose of opening the file in binary mode if we're immediately going to decode it?

Reply:
Oops, fixed